{{ cookiecutter.hosting_github_repo_name }}/web/public/index.html

```diff
@@ -1,43 +1,62 @@
+    </title>
+    <meta
+      name="title"
+      content="Council Data Project | {{ cookiecutter.municipality }} – Searchable city council videos, 
+      transcripts, voting records, and legislation."
+    />
+    <meta
+      name="description"
+      content="City council videos, transcripts, voting records, and legislation all searchable under 
+      a single website."
+    />
     <meta charset="utf-8" />
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+
+    <!-- Open Graph / Facebook -->
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="{{ cookiecutter.hosting_web_app_address }}" />
     <meta
-      name="description"
-      content="CDP - {{ cookiecutter.municipality }}"
+      property="og:title"
+      content="Council Data Project | {{ cookiecutter.municipality }} – Searchable city council videos, 
+      transcripts, voting records, and legislation."
     />
-    <link rel="apple-touch-icon" href="favicon.png" />
-    <!--
-      manifest.json provides metadata used when your web app is installed on a
-      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
-    -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
+    <meta
+      property="og:description"
+      content="City council videos, transcripts, voting records, and legislation all searchable under 
+      a single website."
+    />
+    <meta property="og:image" content="%PUBLIC_URL%/cdp-og-seo.png" />
 
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>CDP - {{ cookiecutter.municipality }}</title>
+    <!-- Twitter -->
+    <meta property="twitter:card" content="summary_large_image" />
+    <meta property="twitter:url" content="{{ cookiecutter.hosting_web_app_address }}" />
+    <meta
+      property="twitter:title"
+      content="Council Data Project | {{ cookiecutter.municipality }} – Searchable city council videos, 
+      transcripts, voting records, and legislation."
+    />
+    <meta
+      property="twitter:description"
+      content="City council videos, transcripts, voting records, and legislation all searchable under 
+      a single website."
+    />
+    <meta
+      property="twitter:image"
+      content="%PUBLIC_URL%/cdp-twitter-seo.png"
```

Comment:
Are the twitter seo and the og seo files different at all? No chance on moving them to cdp-design?

Reply:
Yeah, they slightly differ in resolution (OG == 1200x630, Twitter == 1200x600). 😪

Also, I do plan on making a PR to `cdp-design` (just haven't gotten to it yet), but since the images are only ever referenced from `index.html` I was assuming there would be no way to import the images from an NPM package like you can in JS correct?

Reply:
Hmmmm yea I guess they will just have to live here as well. Oh well.

Reply:
Looks like you can [ship](https://github.com/webpack/webpack/issues/7353) images from an npm package, though I don't know how it would be consumed in the index.html file.

I think it's fine for this PR to have the images here. If we ever need the images elsewhere, we can add them to cdp-design.